### PR TITLE
Utilities module

### PR DESCRIPTION
**Pull request by Martmists-GH**

### CODE REVIEW

File: essentials-module-utilities/src/main/kotlin/io/github/nyliummc/essentials/commands/FlightCommand.kt [hunk 1]
```diff
@@ -26,30 +26,27 @@ package io.github.nyliummc.essentials.commands
...
-    private val abilitySource = Pal.getAbilitySource("essentials", "flight");
+    val ESSENTIALS_ABILITY_FLY = Pal.getAbilitySource("essentials", "flight");
 
     fun register(dispatcher: CommandDispatcher<ServerCommandSource>) {
         Command.builder(dispatcher) {
             command("flight", "fly") {
                 requires { it.hasPermissionLevel(4) }
 
-                executes (::toggleFlightSelf)
+                executes (FlightCommand::toggleFlightSelf)
```
Comment (MarcusElg) [inline essentials-module-utilities/src/main/kotlin/io/github/nyliummc/essentials/commands/FlightCommand.kt]: Why did you change that?

File: essentials-module-utilities/src/main/kotlin/io/github/nyliummc/essentials/commands/GodCommand.kt [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            }
+        }
+    }
+
+    private fun toggleGodSelf(context: CommandContext<ServerCommandSource>): Int {
+        // Set god
+        this.toggleGod(context.source.player)
+
+        // Send feedback
+        context.source.sendFeedback(
```
Comment (MarcusElg) [inline essentials-module-utilities/src/main/kotlin/io/github/nyliummc/essentials/commands/GodCommand.kt:60]: wrong message :P

File: essentials-module-utilities/src/main/kotlin/io/github/nyliummc/essentials/commands/GodCommand.kt [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    private fun toggleGodOther(context: CommandContext<ServerCommandSource>): Int {
+        // Get player
+        val player = EntityArgumentType.getPlayer(context, "player")
+
+        // Set god
+        this.toggleGod(player)
+
+        // Send feedback
+        context.source.sendFeedback(
+                LiteralText("Successfully toggled flight for ${player.displayName.asString()}"),
```
Comment (MarcusElg) [inline essentials-module-utilities/src/main/kotlin/io/github/nyliummc/essentials/commands/GodCommand.kt]: Wrong message here as well

File: essentials-module-utilities/src/main/kotlin/io/github/nyliummc/essentials/configs/UtilitiesConfig.kt [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.github.nyliummc.essentials.configs
+
+data class UtilitiesConfig(
+        val sleepPercentage: Double
```
Comment (MarcusElg) [inline essentials-module-utilities/src/main/kotlin/io/github/nyliummc/essentials/configs/UtilitiesConfig.kt:28]: Can't this be a float? Having it as a double feels like it takes more space than it needs

File: essentials-module-utilities/src/main/kotlin/io/github/nyliummc/essentials/commands/FlightCommand.kt (hunk 1 above)
Comment (Martmists-GH) [reply essentials-module-utilities/src/main/kotlin/io/github/nyliummc/essentials/commands/FlightCommand.kt]: not quite sure why that happened, will fix

File: essentials-module-utilities/src/main/kotlin/io/github/nyliummc/essentials/configs/UtilitiesConfig.kt (hunk 4 above)
Comment (Martmists-GH) [reply essentials-module-utilities/src/main/kotlin/io/github/nyliummc/essentials/configs/UtilitiesConfig.kt:28]: This really doesn't make a difference honestly. We just use Doubles since we use those everywhere in the current code, and having to call .toDouble numerous times is a pain.
